packages/wallets/src/components/Base/WalletPasswordField/WalletPasswordField.scss

```diff
@@ -45,4 +45,11 @@ $meter-widths: (
             }
         }
     }
+
+    .wallets-icon-button {
+        filter: invert(71%) sepia(6%) saturate(0%) hue-rotate(13deg) brightness(85%) contrast(83%); //#999
```

Comment:
is there a reason for adding extra styles here, instead of changing the icon itself?

Reply:
Valid point 👍  Since these icons are only being used in this component I've updated the icons themselves and optimized them through svgo